mission_control/navigator_missions/nav_missions/start_gate.py

```diff
@@ -172,6 +174,8 @@ def make_ogrid_transform(self):
                                                               [0, 0, 0, 1])
         dx = self.x_upper - self.x_lower
         dy = self.y_upper - self.y_lower
+        _max = max(dx, dy)
+        dx = dy = _max
```

Comment:
Hopefully this will be fixed. I need to get some time to fix the ogrid fusing script